src/app/dataset-view/additional-components/flows-component/flows.component.ts

```diff
@@ -93,6 +97,10 @@ export class FlowsComponent extends FlowsTableProcessingBaseComponent implements
         return !this.overview.metadata.currentTransform && this.datasetBasics.kind === DatasetKind.Derivative;
     }
 
+    public get canRunFlows(): boolean {
+        return this.datasetPermissionsService.shouldRunFlows(this.datasetPermissions);
```

Comment:
Why "should"? Can and should are different words :-)

Reply:
Fixed